server.php

```diff
@@ -126,6 +126,7 @@ function valet_default_site_path($config)
  * Inspects registered path directories, case-sensitive.
  */
 $valetSitePath = null;
+$valetSitePathByDomain = null;
```

Comment:
Can this be done without introducing the additional variable?

Reply:
You are right, it is possible.
I changed my PR but I'm not sure if the code is more readable without this extra variable.

Reply:
I understand your point about readability. It's the eternal tension between complexity, readability, cognitive load, and plain old comments! :)

This is one of the few cases where I feel like I prefer using a brief comment instead of a variable. Partly because I didn't feel like the variable was enough to explain why the two "if" clauses can't be combined (and prevent me from re-making that "optimization" again in the future!)

(And, apologies for the breaking change caused by that PR!)